modules/oe_theme_content_news/oe_theme_content_news.info.yml

```diff
@@ -7,4 +7,4 @@ dependencies:
   - extra_field:extra_field
   - oe_content_news:oe_content_news
   - oe_theme:oe_theme_helper
-
+  - ui_patterns:ui_patterns_field_group
```

Comment:
We need to add a post update that enables this new dependency.

Reply:
Implemented in oe_theme_content_news_post_update_00004()